src/Proto.Actor/Future/Futures.cs

```diff
@@ -33,13 +30,12 @@ internal FutureProcess(ActorSystem system) : base(system)
 
             if (!absent) throw new ProcessNameExistException(name, pid);
 
-            Pid = pid;
+            Pid = pid.WithRequestId(1);
```

Comment:
What happens if this is 0?
micro optimization, 0 is not transmitted in protobuf. saving one field when passing the pid remote

Reply:
The advantage with having requestId 1 is that we can classify it as a single request in the deduplication filter. The disadvantage is ofc that there are more bytes transferred, but a pretty negligible difference..